node/service/src/chain_spec.rs

```diff
@@ -235,6 +235,7 @@ fn polkadot_staging_testnet_config_genesis(wasm_binary: &[u8]) -> polkadot::Gene
 			vesting: vec![],
 		}),
 		pallet_vesting: Some(polkadot::VestingConfig { vesting: vec![] }),
+		pallet_treasury: Some(Default::default()),
```

Comment:
It's strange to me that this is needed. Shouldnt be since it was not needed before.